docs/source/guides/specifying_primitive_options.rst

```diff
@@ -54,6 +54,9 @@ Specifying for Individual Primitives
 Options for individual primitives or groups of primitives are set by the ``primitive_options`` parameter of DFS. This parameter
 maps any desired options to specific primitives. In the case of conflicting options, options set at this level will override
 options set at the entire DFS run level, and the include options will always take priority over their ignore counterparts.
+Using the string primitive name or the primitive type will apply the options to all primitives of the same name. You can
+also set options for a specific instance of a primitive. Note, however, that specifying options for a specific instance will
```

Comment:
If I am understanding correctly, the only way to set the options for a specific instance that are different than the options for the primitive generally is to use the instance as the key. If that is correct, we might want to update this a bit for clarity. Maybe change `...also set options for a specific instance of a primitive.` to `...also set options for a specific instance of a primitive by using the primitive instance as a key in the ``primitive_options`` dictionary.`